gatsby/src/layouts/default-layout.tsx

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
 import classnames from 'classnames';
+
+import Header from '@/components/header';
 import Footer from '@/components/footer';
 
 import styles from './default-layout.module.scss';
 
 const DefaultLayout: React.FC = ({ children }) => {
   return (
     <div className={classnames('body__wrapper', styles.wrapper)}>
+      <Header
+        navItems={[
+          { label: 'Úvod', href: '#' },
+          { label: 'Vládní opatření', href: '#', active: true },
```

Comment:
how do we gonna detect active element?

Reply:
https://github.com/cesko-digital/covid.gov.cz/pull/55/files#r516172908